### PR TITLE
DEV2 - search vers 0.1.8

### PR DESCRIPTION
**Pull request by pasquy73**

### CODE REVIEW

**Comment by DemetrixDome** [comment]
Hello, could you please open a ticket for this PR and add a description please in order to approve and merge it?

Many thanks.

**Review by bialask-daiteap — APPROVED** [review]

File: ionos_dev2/search-engine/search/deployment.yaml [hunk 1]
```diff
@@ -17,7 +17,7 @@ spec:
     spec:
       containers:
         - name: search
-          image: production.eng.it:8433/dome/search:0.1.7
+          image: production.eng.it:8433/dome/search:0.1.8
           imagePullPolicy: IfNotPresent
```
Comment (bialask-daiteap) [inline ionos_dev2/search-engine/search/deployment.yaml:21]: Please rebase the changes and squash the commits

**Review by bialask-daiteap — CHANGES_REQUESTED** [review]

**Comment by pasquy73** [comment]
I did it, but I'm not sure it is correct!

**Comment by bialask-daiteap** [comment]
> I did it, but I'm not sure it is correct!

It's surely not. Now there are even more commits - 7 instead of 1.

**Comment by pasquy73** [comment]
I don't know how to do this!

**Comment by bialask-daiteap** [comment]
> I don't know how to do this!

image

**Comment by bialask-daiteap** [comment]
In intellij you just righclick on commit from which you want to start rebasing.
And then "Interactively Rebase from here".
But since you did re-merge from main (which we requested not to do iin the past)  in the past that might not really work. If it wont' work, I suggest to prepare new MR and avoid re-merging from main in the future. Please use rebasing approach instead

**Pull request closed without merging**